src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -42,15 +49,12 @@ export default class Mutations extends React.Component<
         );
     }
 
+    @computed get selectedTranscripId() {
```

Comment:
to fix the error above, try removing @computed here.  it should not be necessary.

Reply:
@alisman I removed the `@computed` but still got the errors